ubuntu/entrypoint.sh

```diff
@@ -91,6 +173,7 @@ fi
 unload_modules rpcrdma rdma_cm
 exit_on_error start_driver
 mount_rootfs
+fix_guid_for_vfs
```

Comment:
nit: please add a comment explaining why we need to fix this for posterity

Reply:
done